testing/test_basic.py

```diff
@@ -108,12 +109,13 @@ def test_MAIN():
 def test_blocon_in_hook(testdir):
     testdir.makeconftest("""
         import pytest
+        import pytest_twisted as pt
```

Comment:
I don't like this but without it `pytest_twisted` is detected as an unknown pytest hook...  :|

```
INTERNALERROR> PluginValidationError: unknown hook 'pytest_twisted' in plugin <module 'conftest' from '/tmp/pytest-of-altendky/pytest-199/testdir/test_blocon_in_hook0/conftest.py'>
```

Reply:
Thanks @altendky. It's ok. I think so.
But if you don't like this, you can try to do smth like:
```python
def pytest_addhooks(pluginmanager):
    pytest.blockon = blockon
    ...
```

Reply:
I think the point is they don't want that.  I thought it was kind of weird when I first saw it, trying to mash all these things into the one scope... But, what I didn't like was specifically that importing pytest_twisted (without `as pt`) caused the above error. But that's just an issue with the module name that happened to become apparent now. Maybe I should raise that question with pytest itself. Might make sense to ignore unknown hooks that are module names of plugins.

Reply:
> But, what I didn't like was specifically that importing pytest_twisted (without as pt) caused the above error. But that's just an issue with the module name that happened to become apparent now. Maybe I should raise that question with pytest itself.

Hi guys dropping in real quick. 😁 

Which pytest and pluggy are you using? This issue was fixed in pytest-dev/pluggy#17 IIRC.